spec/moneyTrackerSpec.js

```diff
@@ -0,0 +1,27 @@
+describe("Moneytracker", function() {
+
+  var moneytracker;
+
+  beforeEach(function() {
+    moneytracker = new Moneytracker();
+  });
+
+  describe("#printStatement", function() {
```

Comment:
Could you mock Statement to isolate these tests from Statement.  This would make them simpler to write. It would also make them less likely to break if Statement changes.